app/src/main/java/mozilla/lockbox/view/FxALoginFragment.kt

```diff
@@ -83,4 +82,9 @@ class FxALoginFragment : BackableFragment(), FxALoginView {
 
     override val skipFxAClicks: Observable<Unit>?
         get() = skipFxA?.clicks()
+
+    override fun onDestroyView() {
```

Comment:
This should already be happening in `Fragment`, the super class of `BackableFragment`